modules/indexer/code/bleve.go

```diff
@@ -333,13 +378,35 @@ func (b *BleveIndexer) Search(repoIDs []int64, keyword string, page, pageSize in
 				endIndex = locationEnd
 			}
 		}
+		language := hit.Fields["Language"].(string)
+		var updatedUnix timeutil.TimeStamp
+		if t, err := time.Parse(time.RFC3339, hit.Fields["UpdatedAt"].(string)); err == nil {
+			updatedUnix = timeutil.TimeStamp(t.Unix())
+		}
 		searchResults[i] = &SearchResult{
-			RepoID:     int64(hit.Fields["RepoID"].(float64)),
-			StartIndex: startIndex,
-			EndIndex:   endIndex,
-			Filename:   filenameOfIndexerID(hit.ID),
-			Content:    hit.Fields["Content"].(string),
+			RepoID:      int64(hit.Fields["RepoID"].(float64)),
+			StartIndex:  startIndex,
+			EndIndex:    endIndex,
+			Filename:    filenameOfIndexerID(hit.ID),
+			Content:     hit.Fields["Content"].(string),
+			CommitID:    hit.Fields["CommitID"].(string),
```

Comment:
Since only the latest commit is held on the indexer, I guess this is for consistency?

Reply:
It is to be sure to point correct file as when you search new code might be not yet indexed and you could be pointed out to non-existing file for example. GitHub also does it this way imho